homeassistant/helpers/restore_state.py

```diff
@@ -48,13 +48,15 @@ def remove_cache(event):
 @asyncio.coroutine
 def async_get_last_state(hass, entity_id: str):
     """Helper to restore state."""
-    if (_RECORDER not in hass.config.components or
```

Comment:
We shouldn't have done this as we now will allow restoring from cache if we are running?